src/pages/LandingPage/LandingPage.tsx

```diff
@@ -23,10 +22,10 @@ const LandingPage = () => {
             <NimbusLogo />
           </XStack>
           <YStack style={{ width: '100%', margin: '30vh 0 4vh' }}>
-            <Title color="$textPrimary">
+            <Text size={27} weight={'semibold'}>
```

Comment:
Is there maybe a token we should use here for this font size?

Reply:
The text component expects number